lib/blueprinter/field.rb

```diff
@@ -10,6 +10,9 @@ def initialize(method, name, extractor, blueprint, options = {})
   end
 
   def extract(object, local_options)
+    # If this is an association field, merge the association field options in
+    # to the render options
+    local_options = local_options.merge(options[:options]) if association? && options[:options]
```

Comment:
Why don't we put this in the `AssociationExtractor`, since this is related to associations?